exercises/resistor-color/resistor_color_test.exs

```diff
@@ -0,0 +1,37 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("resistor_color.exs", __DIR__)
+end
```

Comment:
```suggesstion
```

no longer neccessary in the new mix project style

Reply:
@neenjaw removed!